cogboard-webapp/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_PORT=8092
+REACT_APP_WS_PORT=9001
```

Comment:
I don't think you should've committed this file here.

Reply:
removed